examples/chart/teleport-cluster/values.yaml

```diff
@@ -179,6 +179,12 @@ annotations:
   # Annotations for the ServiceAccount object
   serviceAccount: {}
 
+# Options for the Teleport service
+service:
+  # Additional entries here will be added to the service spec.
+  spec: {}
```

Comment:
This doesn't match the entry in the service spec - i.e. if you specify `service.spec.somethingElse` here, it won't be set, as the service.yaml is only looking for `service.spec.loadBalancerIP`. Please change either the values.yaml to only support `service.spec.loadBalancerIP`, or change service.yaml to check for `service.spec` being set and pass the entire object through.

Reply:
@webvictim Something like this:
```yaml
spec:
  {{- $type := default "LoadBalancer" .Values.service.type }}
  type: {{- $type }}
  {{- with .Values.service.spec }}
  {{- toYaml . | nindent 4 }}
  {{- end }}
  ports:
```

Reply:
Yep!